src/models/methods/subscribe.ts

```diff
@@ -33,16 +33,16 @@ interface BaseStream {
   type: string
 }
 
-export interface LedgerStream extends BaseStream {
```

Comment:
why does this not extend baseStream anymore?

Reply:
This does not extend baseStream anymore because I wanted to make the `type` property optional and `type` is the only field in baseStream. (See below for why I wanted to make `type` optional)